quickstart/index.md

```diff
@@ -74,13 +74,13 @@ The API is easy to use. From a CoreOS machine, you can simply use curl to set an
 Set a key `message` with value `Hello world`:
 
 ```sh
-curl -L http://127.0.0.1:4001/v1/keys/message -d value="Hello world"
+curl -L http://127.0.0.1:4001/v2/keys/message -d value="Hello world"
```

Comment:
It misses '-XPUT'
reference: https://github.com/coreos/etcd/blob/release-0.4/Documentation/api.md#setting-the-value-of-a-key